bitnami/dataplatform-bp2/Chart.yaml

```diff
@@ -0,0 +1,61 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.0
```

Comment:
Why `1.0.0`? Isn't this similar to the "common" chart where the "appVersion" and the "version" should be the same?

Reply:
In this case the appversion and the chart version can be different, checked with OCTO